Assets/Scripts/Enemies/FlyLarvae.cs

```diff
@@ -0,0 +1,86 @@
+﻿/*
```

Comment:
This classed could derive from the  bullet class if its something that the player should be able to shoot at, but if its done it should be done after the bullets get upgraded to differentiate between player and enemy bullets/projectiles